util/Order.cpp

```diff
@@ -237,6 +240,19 @@ void NewFleetOrder::ExecuteImpl() const {
 
     system->FleetsInsertedSignal(created_fleets);
     system->StateChangedSignal();
+
+    // signal change to fleet states
```

Comment:
this comment seems to not fully summarize what the following block of code does...